index.html

```diff
@@ -44,22 +44,21 @@
       depends-on="mgt-teams-provider"
     ></mgt-msal-provider> -->
 
-    <!-- <mgt-msal2-provider
```

Comment:
Is it intentional to pivot this page to use a live msal-2-provider rather than the mock-provider?

Reply:
Not intentional. Was for testing purposes, but should be removed.